packages/features/ee/workflows/lib/reminders/reminderScheduler.ts

```diff
@@ -54,6 +54,8 @@ const processWorkflowStep = async (
     seatReferenceUid,
   }: ProcessWorkflowStepParams
 ) => {
+  if (!step.safe) return;
```

Comment:
in `workflows/utils` `scheduleBookingReminders` we schedule reminders without the `reminderScheduler` and instead call `scheduleEmailReminder` from `emailReminderManager` directly. This is for schedule reminders when an workflow is edited

Reply:
Is there somewhere else I can stop the processing of workflows? This should encompass email and SMS workflows.